text_formatter/exceptions.py

```diff
@@ -0,0 +1,11 @@
+# exceptions.py - Python exceptions for everything at "text_formatter"
+#
+# This file contains many, many exceptions (subclasses of the
+# standard "Exception") and related functions.
+
+class InvalidString(Exception):
```

Comment:
I started with an exception named `InvalidString`. I want to use it on `text_formatter.justify` (#1).

Reply:
Anyway, we can't work with only one exception. We need more files or functions to make this file possible.

I can't generate this file because:

**1. By now, there is no such function taking an exception `InvalidString`, basically because its related function is not complete (#1).**

**2. Even when we have completed `text_formatter.justify`, we'll need more places to call this file and its exceptions. If we don't, this file will only take space. We must make `text_formatter` to be bigger (#7).**

Reply:
#27 wants to take this exception, so we can use it now.